### ISSUE TITLE
[Feature] Can you add the Import / Export config file button

### ISSUE THREAD

**Issue by Marco-Wagonite** [dinogomez/genzai#3]

**Comment by dinogomez** [comment]
Currently, the save config button will save the current settings to Genzai's folder location in JSON format. I can modify it so that you can rename the JSON file on save. To import a config file, just drop it in the same folder as Genzai and restart the app. This process could be improved. I'll see what I can do.

**Comment by Marco-Wagonite** [comment]
But I want to have the Json file on the custom path so I can save that file to import it when I want to redownload the program.

**Comment by dinogomez** [comment]
@Marco-Wagonite  yup I can work on this, just been busy lately. Will do this when I find the time

**Comment by Marco-Wagonite** [comment]
@dinogomez Also can you make it run in the background/ startup with window
